app/experimenter/experiments/models.py

```diff
@@ -100,6 +101,10 @@ class Experiment(ExperimentConstants, models.Model):
     client_matching = models.TextField(
         default=ExperimentConstants.CLIENT_MATCHING_DEFAULT, blank=True
     )
+    locales = models.ManyToManyField(Locale)
+    all_locales = models.BooleanField(default=False)
```

Comment:
Why can't you just infer 'all locales' if locales is empty, instead of storing it on a separate field?

Reply:
We could! I'd be up for that. 

In Delivery Console + Normandy we only store the chosen locales. No "All" boolean choice. But it's impossible to record if the user has made a conscious choice to address it. Kinda like those "[ ] Do you agree? {SAVE CHANGE (button)}" checkboxes. Your browser won't send `agree=false` if the user did *not* check the checkbox. 

Can we assume that the user has made address the choice if they chose nothing? 
I'm generally referring to https://github.com/mozilla/experimenter/issues/971#issuecomment-468835971
If a user never clicks the "Population _Edit_" (e.g. `/experiments/my-slug/edit-variants/`) and then "Save Draft" there, how do you know if the user wants All locales?

Reply:
@peterbe By default all studies target all locales and geos, ie the absence of any specification in the experiment design means the absence of any filtering in the recipe/JEXL and therefore by default all locales/geos.  

This is well understood as part of the overall experiment workflow, and the current interface provides a box where experiment designers can choose to optionally enter any locale/geo filtering if they want or just leave it blank for no filtering.

So I think if we make the two form defaults default to All, and that by default stores no M2M relations, and when we send to Normandy the same is true, no filtering is defined, I think that should match the current workflow.

@shell1 Does that sound right to you?